scripts/installscripts/buster-install-default.sh

```diff
@@ -1192,6 +1192,40 @@ finish_installation() {
             done
     esac
 
+    echo "If you are using a sound card, connect it to your RPi."
+    # Use -e to display response of user in the logfile
+    read -e -r -p "Have you connected your sound card [Y/n] " response
+    case "$response" in
+        [nN][oO]|[nN])
+            ;;
+        *)
+            echo  'Please select the sound card you want to use'
+            options=("Hifiberry" "Pimoroni Pirate Audio" "Manual configuration")
+            select opt in "${options[@]}"; do
+                case $opt in
+                    "Hifiberry")
```

Comment:
Separate for amp2 and miniamp by passing parameter to script, see differences of amp2 and minicamp here: https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1198#issuecomment-750757106